.github/workflows/ci.yml

```diff
@@ -79,6 +79,43 @@ jobs:
       with:
         report_paths: '**/build/test-results/test/TEST-*.xml'
 
+  unit_test_cloud:
```

Comment:
I wonder if this should just be a step in one of the other jobs, and if so, which one? Having this as a separate job means even if it fails it's not going to block a merge because we can't require it because it only works on non-forks.

Reply:
Could you just move the `if` to ` Run cloud test`?

Reply:
Yes, though there's may be a lot of extra stuff for no reason that is tested elsewhere. But I don't mind moving it.

Reply:
What extra tests would be run?

Reply:
No extra tests, extra stuff. The job has 5 steps, but if I put the if around just 1, 4 other steps are running unnecessarily. But I can do that if we want that. Or I can move "Run cloud test" under one of the other jobs.

Reply:
I think it is fine so we can make it block on merge. The other steps are very quick.

Reply:
:+1: Will do that (I'm assuming you're talking about keeping this as a separate job as opposed to just a separate step of an existing job).

Reply:
Done